builder/pipeline_test.cc

```diff
@@ -1259,12 +1265,26 @@ TEST(pipeline, constant) {
   const int W = 20;
   const int H = 10;
 
-  buffer<short, 2> constant_buf({W, H});
-  init_random(constant_buf);
+  auto* dims = SLINKY_ALLOCA(dim, 2);
```

Comment:
`slinky::dim dims[2]`, no need to use alloca for constant size

Reply:
Oh, this is also better because it would call constructors to initialize

Reply:
D'oh